### ISSUE TITLE
Logged In Tabbing tests fail depending on test order

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#4856]
E2E tests should work independently (ie. not rely on data set by other tests) so that proceeding tests don't fail if a preceeding test changes the user data.

**Comment by kacperknapik** [comment]
Fixed with a new `.pressTab()` custom method.
